packages/vuetify/src/components/VCheckbox/VSimpleCheckbox.ts

```diff
@@ -60,6 +60,7 @@ export default Vue.extend({
       },
     }), icon))
 
+    Vue.directive('ripple', ripple)
```

Comment:
I could be misunderstanding - but are you globally importing the ripple directive inside a component?